CHANGELOG.md

```diff
@@ -3,8 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
-- 
-
+- Reverted PR https://github.com/Cotya/magento-composer-installer/pull/34 in favor of a simpler solution without breaking BC
```

Comment:
Seems that markdown does fully work when viewing MD _files_, so, would you change these to `PR [#34](https://github.com/Cotya/magento-composer-installer/pull/34)`? Tedious stuff, but it'll be done once, and then everybody will follow.